packages/aws-cdk-lib/aws-sns/lib/topic.ts

```diff
@@ -86,6 +86,18 @@ export interface TopicProps {
    * @default 1
    */
   readonly signatureVersion?: string;
+
+  /**
+   * Tracing mode of an Amazon SNS topic.
+   *
+   * If PassThrough, the topic passes trace headers received from the Amazon SNS publisher to its subscription.
+   * If set to Active, Amazon SNS will vend X-Ray segment data to topic owner account if the sampled flag in the tracing header is true.
+   *
+   * @see https://docs.aws.amazon.com/sns/latest/dg/sns-active-tracing.html.
+   *
+   * @default PassThrough
+   */
+  readonly tracingConfig?: string;
```

Comment:
An enumeration would be preferred here, since only two values are acceptable. Although I'm not sure why it's not properly documented in the [CloudFormation docs](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-sns-topic.html#cfn-sns-topic-tracingconfig), there should be a list of "Valid Values"

Reply:
fixed to enum.